Makefile

```diff
@@ -18,8 +18,22 @@ GENERATED_FILES := $(UNIT_FILES)
 UNIT_FILES += kata-containers.target
 endif
 
+VERSION := $(shell grep -v ^\# ./VERSION)
+COMMIT_NO := $(shell git rev-parse HEAD 2> /dev/null || true)
+COMMIT := $(if $(shell git status --porcelain --untracked-files=no),${COMMIT_NO}-dirty,${COMMIT_NO})
+
+DESCRIBE := $(shell git describe 2> /dev/null || true)
+DESCRIBE_DIRTY := $(if $(shell git status --porcelain --untracked-files=no 2> /dev/null),${DESCRIBE}-dirty,${DESCRIBE})
+GIT_COMMIT := $(shell git rev-parse HEAD 2>/dev/null)
+ifneq ($(GIT_COMMIT),)
+VERSION_COMMIT := $(VERSION)-$(GIT_COMMIT)
```

Comment:
can you add an `else` statement to fallback if GIT_COMMIT is empty. And assign VERSION_COMMIT=$(VERSION)-known or just VERSION_COMMIT=$(VERSION)